Formula/yugabyte.rb

```diff
@@ -1,17 +1,21 @@
 class Yugabyte < Formula
   desc "High-performance distributed SQL database Yugabyte DB"
   homepage "https://yugabyte.com"
-  url "https://downloads.yugabyte.com/yugabyte-2.0.5.2-darwin.tar.gz"
-  sha256 "8d9b939fe10ba4116c569e27b5b177e9249e448a369dc92b42f5f4a41fa5c8b6"
+  url "https://downloads.yugabyte.com/yugabyte-2.0.6.0-darwin.tar.gz"
+  sha256 "96c23560845647c3dbff4e1923f4a5caec5fb34ccb2e50fccee4c19d5671f3fc"
   depends_on :java => "1.8"
```

Comment:
we also need the python dependency here

Reply:
mac os come with python default. still, if we need some specific version then we can define it.

Reply:
we add a python dep for the client package, so for consistency, it would be good to have it here too.

Reply:
ok